index.html

```diff
@@ -733,6 +733,12 @@ <h3>BitstringStatusListEntry</h3>
 credential.
         </p>
 
+        <p class="note">
+It is important to note that the statusListIndex is the only link between
```

Comment:
Some formatting fixes (I think) here and below.

```suggestion
It is important to note that `statusListIndex` is the only link between
```